src/main/java/com/stripe/exception/MissingRequiredParameter.java

```diff
@@ -0,0 +1,16 @@
+package com.stripe.exception;
+
+public class MissingRequiredParameter extends Exception {
```

Comment:
I have mixed feelings about introducing a new exception class for this. The existing pattern in the library is to throw `InvalidRequestException` when there is a problem with the arguments. This isn't great because it conflates two different types of issues (client-side or server-side) in a single exception class, but it's pretty consistent right now.

I'd be very much in favor of having distinct exceptions, but only if we're consistent about it, which would require some BC incompatible changes.

@brandur-stripe @remi-stripe Any thoughts on the above?

Reply:
I agree with you that we should keep the exceptions list as is for now until we make this in all client-side libraries. It feels like a normal "invalid request" to me.

Reply:
+1. It's not ideal, but we should optimize for consistency, especially where exceptions are concerned in a language like Java that has checked exceptions (existing users might have to go and start decorating Stripe-related method signatures with this new exception type).

Reply:
It's not great b/c `InvalidRequestException` has things like `statusCode`,  `requestId` etc. I guess it's fine to set them to `null`, but doesn't feel super clean.

Reply:
I've changed it to use `InvalidRequestException`.